King-Benx/WeConnect-Android-Java
# Ft create map to endpoints

#### What does this PR do?
This PR delivers an interface that maps to the API endpoints

#### Description of Task to be completed?
The interface creates maps to the API endpoints.

#### How should this be manually tested?
1. Clone the repository onto your machine and build the app. 
2. Run ./gradlew build
3. Check the services/WeConnectService and view the interface

#### Background context
NONE
